File: solutions/961_n-repeated-element-in-size-2n-array/Solution.java
package main;

public class Solution0_java {
    public static int repeatedNTimes(int[] a) {   
        for (int i = 0; i < a.length; ++i) {
            for (int j = 1; j < 4 && i + j < a.length; ++j) {
                if (a[i] == a[i + j]) {
                    return a[i];
                }
            }
        }
        throw new IllegalArgumentException();
    }
}

File: solutions/961_n-repeated-element-in-size-2n-array/Solution.kt
package main

class Solution0_kotlin {
    companion object {
        @JvmStatic
        fun repeatedNTimes(a: IntArray): Int {
            for (i in 0..a.size - 1) {
                for (j in 1..3) {
                    if (i + j >= a.size)
                        continue
                    if (a[i] == a[i + j])
                        return a[i]
                }
            }
            throw IllegalArgumentException()
        }
    }
}
